src/app/shared/services/snack-bar.service.ts

```diff
@@ -0,0 +1,31 @@
+import { inject, Injectable } from '@angular/core';
+import {
+  MatSnackBar,
+  MatSnackBarHorizontalPosition,
+  MatSnackBarVerticalPosition,
+} from '@angular/material/snack-bar';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class SnackBarService {
+  private snackBar = inject(MatSnackBar);
+  private horizontalPosition: MatSnackBarHorizontalPosition = 'center';
+  private verticalPosition: MatSnackBarVerticalPosition = 'top';
+
+  openSnackBar(
+    message: string,
+    duration = 3000,
+    action = 'Zamknij',
+    horizontalPosition = this.horizontalPosition,
+    verticalPosition = this.verticalPosition,
+    panelClass = ['red-snackbar']
```

Comment:
jak mamy tak dużo parametrów funkcji to zróbmy z nich obiekt konfiguracyjny tak by wywoływać tę metodę w taki sposób, gdzie config jest oczywiście pcjonalny bo ma domyślne wartości `openSnackBar(message, config)`